src/rno-g-acq.c

```diff
@@ -1818,11 +1815,16 @@ static int initial_setup()
   }
   pthread_rwlock_init(&ds_lock,NULL); 
 
-
-
   //initialize the radiant lock
   pthread_rwlock_init(&radiant_lock,NULL); 
 
+  // When it is time to do a bias scan record the timing before setting up the radiant
+  if (cfg.radiant.bias_scan.enable_bias_scan && ((cfg.radiant.bias_scan.skip_runs < 2) ||
```

Comment:
In principle we could make these separate config options, though it probably does make sense to tie to bias scan for now.

Reply:
Yes for now I think this is fine and we can add it in the future